share/cmake/modules/Findyaml-cpp.cmake

```diff
@@ -118,51 +124,35 @@ if(NOT OCIO_INSTALL_EXT_PACKAGES STREQUAL ALL)
         if(PC_yaml-cpp_FOUND)
             set(yaml-cpp_VERSION "${PC_yaml-cpp_VERSION}")
         endif()
+
+        # Create target
```

Comment:
Ideally I would stick with the way it's done in other Find scripts, that is at the end, if only to have consistency and allow for easier read and refactoring later if needed.

Reply:
That doesn't make any sense scoping wise. You always need to create the target when you're in this branch and never when you're in the other one above.

Reply:
Yes and that's handled in all other Find* scripts we have with use with conditionals at the end of the script.

Reply:
Updated now.